tests/codegen/ret_value.rs

```diff
@@ -0,0 +1,12 @@
+#![feature(no_core)]
+#![no_core]
+#![no_main]
+
+extern crate mini_core;
+
+// expect three int params
+// CHECK: 42
+#[no_mangle]
+pub fn main() -> i32 {
```

Comment:
```rust
// CHECK-LABEL: main
// CHECK: return 42;
#[no_mangle]
pub fn main() -> i32 {
```

It is usually always good to use ` CHECK-LABEL`. And I assume you want to check that it actually returns 42, not just that it's somewhere in the code :)